sufia-models/lib/sufia/models/engine.rb

```diff
@@ -30,6 +28,14 @@ class Engine < ::Rails::Engine
       config.max_notifications_for_dashboard = 5
       config.activity_to_show_default_seconds_since_now = 24*60*60
 
+      # Noid identifiers
+      config.enable_noids = true
+      config.noid_template = '.reeddeeddk'
+      config.minter_statefile = '/tmp/minter-state'
```

Comment:
This PR does not change the default `minter_statefile`; it just reorganizes Noid-related configs into a contiguous block. I'm reluctant to change the default value because `/tmp` is a well known location that is likely to exist in most environments and that is writeable by all users. If you have a different default location to recommend, I am cool with that, but I'm thinking you should do that in a separate PR since it's unrelated to what I'm trying to do with this PR which is to update to AF::Noid 1.0 and retain the previous treeifying behavior.